### PR TITLE
Bare bones answer picker

### PR DESCRIPTION
**Pull request by demoleas21**
## Summary

Created bare bones answer picker view with label to display choice
- ~~Need to redo elements programmatically~~
- ~~Need to reset answer label after moving to the next question~~
- ~~Need to store each answer for usage statistics~~

### CODE REVIEW

File: app-steady/app-steady/Base.lproj/LaunchScreen.storyboard [hunk 1]
```diff
@@ -15,7 +16,6 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <animations/>
```
Comment (DanielHabib) [inline app-steady/app-steady/Base.lproj/LaunchScreen.storyboard:18]: Did you add something to the storyboard?

**Comment by DanielHabib** [comment]
Seems alright, but lets avoid using the storyboard editor. 
All of this can be implemented programatically

File: app-steady/app-steady/FirstViewController.swift [hunk 2]
```diff
@@ -22,7 +19,21 @@ class FirstViewController: UIViewController,UIPickerViewDataSource,UIPickerViewD
...
         questionLabel.text = mainQuestion.askQuestion
+        
+        let answerLabel = UILabel(frame: CGRectMake(0, 100, 200, 21))
+        answerLabel.textAlignment = NSTextAlignment.Center
+        answerLabel.text = ""
+        
+        let nextQuestionButton = UIButton(type: UIButtonType.System)
+        nextQuestionButton.setTitle("Next Question", forState: UIControlState.Normal)
+        nextQuestionButton.frame = CGRectMake(0, 200, 200, 21)
+        nextQuestionButton.addTarget(self, action: #selector(FirstViewController.nexQuestionButtonPressed), forControlEvents: UIControlEvents.TouchUpInside)
```
Comment (demoleas21) [inline app-steady/app-steady/FirstViewController.swift]: Error Here

File: app-steady/app-steady/FirstViewController.swift [hunk 3]
```diff
@@ -32,10 +43,11 @@ class FirstViewController: UIViewController,UIPickerViewDataSource,UIPickerViewD
...
     
-    @IBAction func nextQuestionButtonPressed(sender: AnyObject) {
-        self.mainQuestion = nextQuestion()
-        self.questionLabel.text = mainQuestion.askQuestion
-        self.answerLabel.text = ""
+    func nextQuestionButtonPressed(sender: UIButton!) {
+        mainQuestion = nextQuestion()
+        questionLabel.text = mainQuestion.askQuestion
+        NSLog(answerLabel.text)
+        answerLabel.text = ""
```
Comment (demoleas21) [inline app-steady/app-steady/FirstViewController.swift]: Error with lines 48, 49, 50

File: app-steady/app-steady/FirstViewController.swift [hunk 4]
```diff
@@ -20,15 +23,13 @@ class FirstViewController: UIViewController,UIPickerViewDataSource,UIPickerViewD
...
         questionLabel.text = mainQuestion.askQuestion
         
-        let answerLabel = UILabel(frame: CGRectMake(0, 100, 200, 21))
         answerLabel.textAlignment = NSTextAlignment.Center
         answerLabel.text = ""
         
-        let nextQuestionButton = UIButton(type: UIButtonType.System)
         nextQuestionButton.setTitle("Next Question", forState: UIControlState.Normal)
         nextQuestionButton.frame = CGRectMake(0, 200, 200, 21)
         nextQuestionButton.addTarget(self, action: #selector(FirstViewController.nexQuestionButtonPressed), forControlEvents: UIControlEvents.TouchUpInside)
```
Comment (demoleas21) [inline app-steady/app-steady/FirstViewController.swift]: Error: `Type 'FirstViewController' has no member 'nextQuestionButtonPressed'`

File: app-steady/app-steady/FirstViewController.swift [hunk 5]
```diff
@@ -11,6 +11,9 @@ import UIKit
 class FirstViewController: UIViewController,UIPickerViewDataSource,UIPickerViewDelegate {
     
     @IBOutlet weak var answerPicker: UIPickerView!
+    var questionLabel = UILabel(frame: CGRectMake(0, 0, 200, 21))
+    var answerLabel = UILabel(frame: CGRectMake(0, 100, 200, 21))
+    let nextQuestionButton = UIButton(type: UIButtonType.System)
```
Comment (demoleas21) [inline app-steady/app-steady/FirstViewController.swift:18]: Fixed previous errors

File: app-steady/app-steady/FirstViewController.swift [hunk 6]
```diff
@@ -8,27 +8,65 @@
...
+        
+        questionLabel.textAlignment = NSTextAlignment.Center
         questionLabel.text = mainQuestion.askQuestion
+        
+        answerLabel.textAlignment = NSTextAlignment.Center
+        answerLabel.text = ""
+        
+        nextQuestionButton.setTitle("Next Question", forState: UIControlState.Normal)
+        nextQuestionButton.frame = CGRectMake(0, 200, 200, 21)
+        nextQuestionButton.addTarget(self, action: #selector(FirstViewController.nexQuestionButtonPressed), forControlEvents: UIControlEvents.TouchUpInside)
```
Comment (DanielHabib) [inline app-steady/app-steady/FirstViewController.swift]: try putting a colon `:` after  `FirstViewController.nextQuestionButtonPressed` like `FirstViewController.nextQuestionButtonPressed:`

File: app-steady/app-steady/FirstViewController.swift [hunk 7]
```diff
@@ -8,27 +8,65 @@
...
+        
+        answerLabel.textAlignment = NSTextAlignment.Center
+        answerLabel.text = ""
+        
+        nextQuestionButton.setTitle("Next Question", forState: UIControlState.Normal)
+        nextQuestionButton.frame = CGRectMake(50, 400, 200, 21)
+        nextQuestionButton.addTarget(self, action: #selector(self.nextQuestionButtonPressed), forControlEvents: UIControlEvents.TouchUpInside)
+        
+        self.view.addSubview(nextQuestionButton)
+        self.view.addSubview(questionLabel)
```
Comment (DanielHabib) [inline app-steady/app-steady/FirstViewController.swift:39]: See how we add the subview here. If you uncomment these lines both will disappear

**Comment by DanielHabib** [comment]
:+1:

**Comment by DanielHabib** [comment]
funny-pokemon-laughing-Jir3toQTWW9Ne

**Pull request merged**